### PR TITLE
Use Rubygems Metadata Presenter

### PR DESCRIPTION
**Pull request by brenetic**
We now publish the Metadata Presenter to Rubygems

### CODE REVIEW

**Review by njseeto — APPROVED** [review]
🚀

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '2.7.2'
 
 gem 'bootsnap', '>= 1.4.2', require: false
-gem 'metadata_presenter', github: 'ministryofjustice/fb-metadata-presenter', branch: 'main'
+gem 'metadata_presenter', '~> 0.1.2'
```
Comment (njseeto) [inline Gemfile:7]: i think this needs to be updated to 0.1.3?

File: Gemfile.lock [hunk 2]
```diff
@@ -223,7 +217,7 @@ DEPENDENCIES
   brakeman
   byebug
   listen (~> 3.3)
-  metadata_presenter!
+  metadata_presenter (~> 0.1.2)
```
Comment (njseeto) [inline Gemfile.lock:220]: similarly here

**Review by njseeto — APPROVED** [review]
Looks good - just need to update the gem to 0.1.3 🚀

**Comment by brenetic** [comment]
thanks @njseeto have updated

**Review by njseeto — APPROVED** [review]
LGTM! 🚀 🥇

**Pull request merged**